apis/bases/core.openstack.org_openstackcontrolplanes.yaml

```diff
@@ -7701,6 +7701,22 @@ spec:
                     - secret
                     type: object
                 type: object
+              lightspeed:
+                properties:
+                  enabled:
+                    default: false
+                    type: boolean
+                  llm_credentials:
+                    type: string
+                  llm_endpoint:
+                    type: string
+                  tlsCert:
+                    type: string
+                required:
+                - llm_credentials
+                - llm_endpoint
+                - tlsCert
```

Comment:
Maybe a silly question. But how one configure the Model, IndexID, etc... ? Is this expected to be configure directly into OLS itself ?

Or perhaps we will add that stuff later ?

Reply:
Thanks @umago! These are good points. I'll be polishing the fields at the end. I think I will include these.